lxd/images.go

```diff
@@ -1294,6 +1295,10 @@ func doImagesGet(ctx context.Context, tx *db.ClusterTx, recursion bool, projectN
 	}
 
 	for _, fingerprint := range fingerprints {
+		if !hasPermission(fingerprint) {
```

Comment:
Does this call out to the open FGA server to check permissions for each image fingerprint? If so, it might be worth filtering these after the initial transaction has completed. (Perhaps we can also call in a batch?)

Reply:
> Does this call out to the open FGA server to check permissions for each image fingerprint? 

No, it doesn't. We gather a list of allowed objects, and perform a check against that list.

Reply:
Ah yes, I misread the `getReBACPermissionChecker` method.